tests/analytics/test_v2_replay_determinism.py

```
@@ -0,0 +1,115 @@
+        request = V2AlignedInputRequest(
+            T=T, symbol=SYMBOL, market_type=MARKET_TYPE, calculation_version=CALC_VERSION,
+            feature_schema_version=1, health_exchanges=(EXCHANGE,), health_metrics=("ohlcv",),
+        )
+        aligned = await load_v2_aligned_inputs(session, request)
+        context = build_v2_context_snapshot(aligned)
+        setup_inputs = await load_compression_breakout_inputs(session, context=context)
+    return aligned, context, setup_inputs
+
+
+def test_two_independent_replay_sessions_produce_identical_facts():
+    async def body(db, _dsn):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `body`

Add return type annotation: `None`

(ANN202)